.travis.yml

```diff
@@ -1,5 +1,43 @@
+sudo: required
+dist: trusty
+
 language: c
 compiler: gcc
 
+before_script:
+- sudo apt-get -qq install valgrind lcov uuid-dev pkg-config zlib1g-dev libffi-dev gettext libpcre3-dev texinfo gtk-doc-tools cppcheck
+- mkdir cor-dependencies
+- pushd cor-dependencies
+- curl -L -O https://github.com/GNOME/glib/archive/2.49.4.tar.gz
+- tar -xvf 2.49.4.tar.gz
+- pushd glib-2.49.4
+- bash autogen.sh --prefix=/usr
```

Comment:
Hi, 

I looked at the `ccache` feature and it seems that it is worth only in the case we recompile (make clean; make). So maybe it will not benefit us. 
Good idea about the `.travis-setup.sh` file,  I will change it and resubmit.